modules/puzzle/src/main/Attempt.scala

```diff
@@ -34,40 +30,34 @@ object Attempt {
     val date = "d"
     val win = "w"
     val time = "t"
```

Comment:
I argue that if and when we come to adding analytics to tactics (and I strongly think that it's something that we should do). Being able to find out which tactics and motifs you have the most trouble finding is highly valuable information.